### ISSUE TITLE
EO violation in Wire?

### ISSUE THREAD

**Issue by fabriciofx** [yegor256/cactoos-http#70]
Into the `Wire` interface we have a method `Input send(Input input) throws IOException`, but it isn't a EO violation? In EO vol 1, Yegor says *"verb methods should return nothing (void)"* and `send()` is a verb. So, should we rename it?

**Comment by 0crat** [comment]
@llorllale[/z](https://www.0crat.com/u/llorllale) please, pay attention to this issue

**Comment by llorllale** [comment]
@fabriciofx fair enough, but neither solutions are correct according to Yegor's book.

**Comment by fabriciofx** [comment]
@llorllale why not?

**Comment by fabriciofx** [comment]
@llorllale WDYT:
```
interface Request extends Input {
}

interface Response extends Input {
}

Response request(Request req) throws IOException
```
